icu4c/source/test/intltest/rbbitst.cpp

```diff
@@ -2789,11 +2797,16 @@ RBBILineMonkey::RBBILineMonkey() :
 
     std::list<std::pair<std::string, UnicodeSet>> partition;
 
-    // TODO(egg): The following two are a workaround for what seems to be an ICU bug.
+    // TODO(egg): The following two workarounds for what seems to be ICU bugs;
+    // with UREGEX_DOTALL (but not UREGEX_MULTILINE):
+    // 1. /.*\u000A/ does not match CR LF;
```

Comment:
I think that's because . matches CR LF, so it won't match just half of that.

Reply:
That may well be the explanation, but even with regex greed, that still feels like a bug: `/(rn|[a-z])*n/` still matches `rn`.